src/SortableContainer/index.js

```diff
@@ -430,7 +445,7 @@ export default function sortableContainer(WrappedComponent, config = {withRef: f
           top: offset.top + node.offsetTop,
           left: offset.left + node.offsetLeft,
         };
-        if (node.parentNode !== this.container) {
+        if (getOffsetParent(node, this.container) !== this.container) {
```

Comment:
relevant change in index.js
instead of comparing the parent node with the container, use the offsetParent (first positioned node)